fastlane/Fastfile

```diff
@@ -223,6 +223,17 @@ platform :ios do
     github_release(version: version_number)
   end
 
+  desc "Tag current branch with current version number"
+  lane :tag_current_branch do |options|
+    version_number = current_version_number
+    add_git_tag(
```

Comment:
Should we maybe check if the tag already exists not do anything? We could delete the tag and retag, but it's dangerous since we might be releasing the same version multiple times, so I suggest the first option.